addons/godot-xr-tools/functions/movement_sprint.gd

```diff
@@ -162,3 +162,12 @@ func _get_configuration_warning():
 
 	# Call base class
 	return ._get_configuration_warning()
+## Find an [XRToolsMovementSprint] node.
```

Comment:
Could you add a couple of blank lines between these functions - the [gdscript style guide](https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_styleguide.html#blank-lines) suggests this style to clearly separate the functions.